proposals/lip-0058.md

```diff
@@ -704,8 +703,7 @@ This function returns the address of the generator active at the timestamp `time
 
 ```python
 def getGeneratorAtTimestamp(timestamp, currentHeight):
-    elapsedTime = timestamp - genesisTimestamp
-    slotIndex = floor(elapsedTime / BLOCK_TIME) % len(generatorList)
+    slotIndex = floor(timestamp / BLOCK_TIME) % len(generatorList)
```

Comment:
We should avoid using `floor` and instead use integer division (`//`).

Reply:
changed everywhere in LIP 0044 and LIP 0058.